### ISSUE TITLE
"in advance of" should read "newer than"

### ISSUE THREAD

**Issue by twaugh** [fedora-infra/fmn#96]
The notification emails say:

```
A new version of "(package)" has been detected: "(ver)" in advance of "(ver)", packaged as "(pkg)"
```

I read "in advance of" as meaning "prior to", which makes it confusing/awkward to read. Maybe it should say "newer than" instead? Even just ">" would read better, IMHO.

**Comment by relrod** [comment]
+1 for `>`.

**Comment by ralphbean** [comment]
Can do.. **except** the `>` might mess up the display in some HTML contexts... hm.

**Comment by ralphbean** [comment]
So, the place to change this is [right here](https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/blob/develop/fedmsg_meta_fedora_infrastructure/anitya.py#L144-L146) and there are some test cases in that repo that would need to be updated also (for anyone who wants to take this on).

**Comment by ralphbean** [comment]
This was fixed in 684c984118c52126154ad4b03bacf1497635a4b9.
